.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,81 @@
+name: Static Analysis
+on: [push, pull_request]
+jobs:
+  clang-tidy:
+    name: Clang Tidy
+    runs-on: ubuntu-latest
+    container: ubuntu:devel
+    env:
+      DEBIAN_FRONTEND: "noninteractive"
+      inexor_conan_path: "$HOME/.local/bin"
+      inexor_vulkan_sdk: "$HOME/vulkan_sdk/"
+      inexor_vulkan_version: "1.2.131.1"
```

Comment:
Can you take a look into the [ci fix MR](https://github.com/inexorgame/vulkan-renderer/pull/363)? i changed the workflow there, would be nice to reflect it here too.